stress_load/helpers/connection.js

```diff
@@ -0,0 +1,85 @@
+
+const SEND_TX_PAYLOAD = "{\"data\":\"4a65e4bb49565bdcc326365d5c0e2db5dceb37ab29f9f423eeaf7e0ded0b92e0ea2adb5c75692b644c9072f3fea0a1657351986678faadfbbd55ef6c6d74d3268ae83769c94e0ac6e77425930dee7951e727ba70afdb091894ed85f41168025dd512e857fa9d8c589bb4ead030cb3734453f42404f2fea17d49d1327ed5a274c0ea0bc30e761c62708bc14ef2c6fe960fc31763cf1e356ffead744bb5fdbd2f9785a85b8b2a01f3ec62756f0c8bf1ec000de8aaf7d09720d252c2a0a5492693638feb99d4a9924178cc224448c298c67c0822f184e61e398b9397847893d5423907a9409fcac4faec427cd67c0c023ae5bb41c3802935a8120cab7f1d521cfd27669ebce7b1401dacba76575eb95c7c4\",\"hmac\":\"9f535249351da6323104d97e680c5df259768aff718f8952810eb34407c96bc7\",\"iv\":\"8ee50886706bd88f8106fecc93c855e3\"}";
+const SENT_TX_PAYLOAD = "{\"data\":\"1826e47e22eb933f5beed0796d62137be6200f18fa278314ed8090cfc9c4fbe063bc079fcd406dd4c1b72821fa1e0efdc4a4776bceea279f1b9cc4b019a8fd2f1dff18e4a3f74a1d281b42bc94aba187f8dfdfbd964edf9c96d738192a68dd69f9fa430dc2cd57ae392e53ba0fe6958e\",\"hmac\":\"ab86d6953ef1a70996dc6ffeaa1ac10f6f3fc98a1eed4784d6837607ed6688a6\",\"iv\":\"cba85757305f520a02a58fccdee3fd81\"}";
+
+const generateConnectionMessages = () => {
+  const topic1 = generateRandomTopic()
+  const topic2 = generateRandomTopic()
+  const topic3 = generateRandomTopic()
+
+  const {pub: pub1, sub: sub1, ack: ack1} = genertePubSubMessages(topic1, CONNECTION_PAYLOAD)
+  const {pub: pub2, sub: sub2, ack: ack2} = genertePubSubMessages(topic2)
+  const {pub: pub3, sub: sub3, ack: ack3} = genertePubSubMessages(topic3, CONFIRM_CONNECTION_PAYLOAD)
+
+ function * generateDAPPConnectionMessages () {
+  //  publish to topic 1
+  yield pub1
+
+  // subscribe to topic 2
+  yield sub2
+
+  // in response to approve connection
+  yield ack2
+ }
+
+ function * generateWalletConnectionMessages () {
+  //  subscribe to topic 1
+  yield sub1
+
+  // subscribe to topic 3
+  yield sub3
+
+  // in response to request connection
+  yield ack1
+
+  // confirm connection
+  yield pub2;
+ }
+
+ function * generateDAPPSendTxMessages () {
+   // publish to topic 3
+   yiled ({...pub3, payload: SEND_TX_PAYLOAD})
+
+   // in response to Confirm TX from wallet
+   yield ack2
+ }
+
+ function * generateWalletSendTxMessages () {
+   // in response to Send TX from dapp
+   yield ack3
+
+   // in response to Confirm TX from wallet
+   yield ({...pub2, payload: SENT_TX_PAYLOAD})
+ }
+
+ const DAPConnectionGen = generateDAPPConnectionMessages()
+ const WalletConnectionGen = generateWalletConnectionMessages()
+ const DAPPTxGen = generateDAPPSendTxMessages()
+ const WalletTxGen = generateDAPPSendTxMessages()
```

Comment:
following [this thread](https://github.com/paraswap/wallet-connect-relay-heroku/pull/3/files#r660003521),

```diff
-const WalletTxGen = generateDAPPSendTxMessages()
+const WalletTxGen = generateWalletSendTxMessages()
```